appveyor.yml

```diff
@@ -4,7 +4,7 @@
       only:
         - master
   version: 1.0.0.{build}
-  image: Visual Studio 2017
+  image: Visual Studio 2015
```

Comment:
As far as I know, the 2017 image does not work properly with Xamarin solutions yet...
(https://ci.appveyor.com/project/MvvmCross/mvvmcross/build/1.0.144)